client/my-sites/theme/index.web.js

```diff
@@ -28,11 +32,35 @@ function addNavigationIfLoggedIn( context, next ) {
 	next();
 }
 
+function setTitleAndSelectSiteIfLoggedIn( context, next ) {
+	const state = context.store.getState();
+	const theme = getCanonicalTheme( state, null, context.params.slug );
+	const themeName = theme.name;
+
+	context.getSiteSelectionHeaderText = () =>
+		translate( 'Select a site to see {{strong}}%(themeName)s{{/strong}} details', {
```

Comment:
How about: Select a site to view **Spearhead**

<img src="https://user-images.githubusercontent.com/10274366/124800941-3b33cf80-df24-11eb-9f2b-781cad2d7073.png" width=200 />

This should match up with the Themes page

<img src="https://user-images.githubusercontent.com/10274366/124800973-471f9180-df24-11eb-85bc-c76f366fbf47.png" width=200px />

Reply:
That's indeed better. I will fix it!